lib/components/Pagination/Pagination.tsx

```diff
@@ -45,18 +45,7 @@ export const Pagination = ({
       type={type}
       className="ds-inline-flex ds-list-none ds-items-center ds-justify-center ds-gap-3"
     >
-      <button
-        onClick={() => onPageChange({ page: 1, pageSize })}
-        className={getClassName({ disabled: isFirstPage })}
-        disabled={isFirstPage}
-        type="button"
-      >
-        <span className="ds-sr-only">{translations.firstPageTriggerLabel}</span>
-        <MdFirstPage size={24} />
-      </button>
-
       <ArkPagination.PrevTrigger className={getClassName({ disabled: isFirstPage })} disabled={isFirstPage}>
-        <span className="ds-sr-only">{translations.prevTriggerLabel}</span>
```

Comment:
No need to explicitly tell this, as the Ark UI already seems to pass the given translation as `aria-label` to the button.

Same for the `nextTriggerLabel`